iOSDemo/DemoViewController.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+import UIKit
+import SweetUIKit
+
+enum DemoItem: Int {
+    case table, edit, collection
+    
+    static var titles: [String] {
+        let allItems: [DemoItem] = [.table, .edit, .collection]
+        return allItems.map{ $0.title }
+    }
+    
+    var title: String {
+        switch self {
+        case .table: return "Table Controller"
+        case .edit: return "Edit Controller"
+        case .collection: return "Collection Controller"
+        }
+    }
+    
+    var viewController: UIViewController {
+        switch self {
+        case .table: return TableController()
```

Comment:
I think is fine if we skip having `TableController` since `DemoViewController` is a TableView demo by itself.